MANIFEST.in

```
@@ -0,0 +1,19 @@
+include LICENSE
+include README.md
+
+recursive-include uxarray *.py
+recursive-exclude * __pycache__
+recursive-exclude * *.py[co]
+exclude .gitignore
+exclude .pre-commit-config.yaml
+exclude .readthedocs.yml
+
+include *.md
+include *.toml
+include *.yaml
```

```suggestion
exclude .pre-commit-config.yaml
exclude .readthedocs.yml

include *.md
include *.toml
```

The `include *.yaml` is overriding your `exclude .pre-commit-config.yaml`. I checked through your `SOURCES.txt` and `exclude .pre-commit-config.yaml` seems to be the only `*.yaml` file you're capturing anyway. I suggest removing the `include *.yaml` line.